### PR TITLE
Add orml token balances to Portfolio

### LINKED ISSUE #160: Show bridget assets in the portfolio
Show the amounts of bridged assets that the connected account holds.
Use the following screenshot as a guidance:

image

### PR DESCRIPTION
**Pull request by gonzamontiel**
Closes #160

### CODE REVIEW

**Comment by ebma** [comment]
@gonzamontiel I changed the logic a bit so that it now uses a typed array which always gets a new entry for each fetched balance (even if it's 0). But otherwise, we might not update the map in case the balance changes to 0 while the user is using the portal. I also changed it to fetch also the balances of the collateral currencies because that might also be interesting to see for users and/or vault operators.

File: src/pages/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -50,43 +49,42 @@ function Dashboard() {
...
 
-    wrappedCurrencies.forEach((currencyId) => {
+    availableCurrencies.forEach((currencyId) => {
       const walletAddress = ss58Format ? getAddressForFormat(walletAccount.address, ss58Format) : walletAccount.address;
       fetchBridgedTokens(walletAddress, currencyId).then((balance) => {
         setAccountTokenBalances((oldBalances) => {
-          const tokenId = currencyToString(currencyId);
-          if (tokenId && balance) {
-            oldBalances[tokenId] = balance;
+          try {
```
Comment (gonzamontiel) [inline src/pages/dashboard/Dashboard.tsx]: I guess this try/catch is not needed, the full currencyToString() is wrapped in a try/catch block already, and right after we are asking for tokenId value making sure it is not null

**Review by gonzamontiel — COMMENTED** [review]
I added a comment and fixed it. I think it's ready.

**Comment by gonzamontiel** [comment]
> I added a comment and fixed it. I think it's ready.

As I'm the owner, I think @ebma you have the final word approving.

File: src/pages/dashboard/Dashboard.tsx (hunk 1 above)
Comment (gonzamontiel) [reply src/pages/dashboard/Dashboard.tsx]: fixed it.

**Review by ebma — APPROVED** [review]
LGTM

**Pull request merged**